backend.native/compiletest/src/main/kotlin/org/jetbrains/kotlin/compiletest/DistProperties.kt

```diff
@@ -0,0 +1,17 @@
+package org.jetbrains.kotlin.compiletest
+
+import java.nio.file.Path
+import java.nio.file.Paths
+
+
+object DistProperties {
+    private val isWindows: Boolean = requireProp("os.name").startsWith("Windows")
+
+    private val dist: Path = Paths.get(requireProp("konan.home"))
+    private val konancDriver = if (isWindows) "konanc.bat" else "konanc"
+    val konanc: Path = dist.resolve("bin/$konancDriver")
+    val lldb: Path = Paths.get("lldb")
```

Comment:
This obviously should use something real, but I don't know at the moment how we are going to provide lldb for testing :) 

Currently, tests are skipped if there's no lldb.